### PR TITLE
Add code comment style

### PR DESCRIPTION
**Pull request by CaitinChen**
@lilin90 @shenli @gingerkidney  PTAL Thanks.

### CODE REVIEW

File: code-comment-style.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+- FIXME
+
+## How to comment?
+
+### Format of a good comment
+
+- Go
+    
+    - Use **//** for a single-line comment and trailing comment
+    - Use **/* ... */** for a block comment (used only when needed)
```
Comment (lilin90) [inline code-comment-style.md]: `/* ... */` cannot be correctly displayed.

File: code-comment-style.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    - Use words and expressions consistently
+        
+        - dead link vs. broken link (Only one of them can appear in a single document)
+    
+    - Do not use lengthy compound words
+
+    - Do not abbreviate unless it is absolutely necessary
+
+    - "we" can be used only when it means the code writer
```
Comment (lilin90) [inline code-comment-style.md]: Need update what "we" means here?

**Comment by CaitinChen** [comment]
This PR will link to the comment style in the tikv repo. It can only be merged when https://github.com/tikv/tikv/pull/4204 (waiting for Brian's review tomorrow) is merged

**Comment by CaitinChen** [comment]
https://github.com/tikv/tikv/pull/4204 is closed.
https://github.com/tikv/tikv/pull/4259 is merged. @lilin90 PTAL

File: code-comment-style.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# Code Comment Style
+
+This document describes the code comment style applied to all PingCAP repositories. When you are to commit, be sure to follow the style to write good code comments.
+
+For the comment style of Rust, the main programming language of TiKV (an important component of the TiDB project), see [the code comment style for the TiKV project](https://github.com/tikv/tikv/blob/master/CODE_COMMENT_STYLE.md).
```
Comment (lilin90) [inline code-comment-style.md]:
```suggestion
For the comment style of Rust, the main programming language of TiKV (an important component of the TiDB cluster), see [the code comment style for the TiKV project](https://github.com/tikv/tikv/blob/master/CODE_COMMENT_STYLE.md).
```

**Comment by CaitinChen** [comment]
@shenli PTAL

**Pull request merged**